examples/particle_rayleigh_taylor_mu10/RTplot.py

```diff
@@ -0,0 +1,74 @@
+# This sript plots up the RMS velocity:
```

Comment:
Let's add a `#!/usr/bin/env python3` to be consistent

Reply:
Also, can you do a `chmod +x  examples/particle_rayleigh_taylor_mu10/RTplot.py` (and then commit), this makes the script executable, so users can just do `./RTplot.py` if they're in the directory.